src/Extensions/CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,6 @@
+# Code of Conduct
```

Comment:
Do not create a separate file here. This is still part of the dotnet-monitor repo so it should be using the files at the root.